healthcheck.go

```diff
@@ -13,33 +17,94 @@ type healthService struct {
 }
 
 type healthConfig struct {
-	appSystemCode string
-	appName       string
-	port          string
+	appDesc                  string
+	appSystemCode            string
+	appName                  string
+	port                     string
+	writerHealthURI          string
+	contentResolverHealthURI string
+	producer                 producer.MessageProducer
+	client                   *http.Client
 }
 
 func newHealthService(config *healthConfig) *healthService {
 	service := &healthService{config: config}
 	service.checks = []health.Check{
-		service.sampleCheck(),
+		service.writerCheck(),
+		service.contentResolverCheck(),
+		service.producerCheck(),
 	}
 	return service
 }
 
-func (service *healthService) sampleCheck() health.Check {
+func (service *healthService) buildHealthCheck() health.HealthCheck {
+	return health.HealthCheck{
+		SystemCode:  service.config.appSystemCode,
+		Name:        service.config.appName,
+		Description: service.config.appDesc,
+		Checks:      service.checks,
+	}
+}
+
+func (service *healthService) writerCheck() health.Check {
+	return health.Check{
+		BusinessImpact:   "Content relationships to packages will not be written / updated",
+		Name:             "Content collection Neo4j writer health check",
+		PanicGuide:       "https://dewey.ft.com/upp-content-collection-rw-neo4j.html",
```

Comment:
We could contruct the panicGuide url from the appSystemCode - if we have that extracted already.